### PR TITLE
Remove deprecated prefix 'inputs' from variable names

### PR DESCRIPTION
**Pull request by rinckm**
According to the [tekton documentation](https://github.com/tektoncd/pipeline/blob/master/docs/variables.md#variables-available-in-a-task) variables for parameters used in tasks are not using the prefix 'inputs'. Variables with this prefix are still working correctly but we should switch to the documented way. 

### Dependency release notes

N/A

### Submitter checklist

- [x] Change has been tested (on a back-end cluster)
- [x] [changelog.yaml] with upgrade notes are prepared and appropriate for the audience affected by the change (users or developer, depending on the change).
- [x] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
  - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [ ] n/a Links to external changelogs added since the last release of our component
- [ ] n/a Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [ ] n/a Check if dependency update affects our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [x] There is at least one approval for the pull request and no outstanding requests for change
- [x] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [x] The Pull Request title is understandable and reflects the changes well
- [x] The Pull Request description is understandable and well documented
- [x] 'Upgrade notes' are documented in changelog.yaml (if required)
- [x] [changelog.yaml] entry for this Pull Request has been added
  - [x] Changelog entry contains all required information

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

File: changelog.yaml [hunk 1]
```diff
@@ -18,7 +18,15 @@
...
   date: TBD
-  
+  changes:
+  - type: internal
+    impact: patch
+    title: Fix clustertask
+    description: >-
+      Fix variables in clustertask to changed structure.
+    warning:
+    upgradeNotes: See upgrade notes of version 0.5.0
```
Comment (anbrsap) [inline changelog.yaml:28]: If you have 0.5.0 already installed, do you need to consider anything? If not, then just leave this field empty here.

Comment (rinckm) [inline changelog.yaml:28]:
```suggestion
    upgradeNotes:

```

Comment (rinckm) [reply changelog.yaml]: Done: https://github.com/SAP/stewardci-core/pull/173/commits/9a3b54b1ee45f6e71824f04398876cd2758ad922

File: changelog.yaml [hunk 2]
```diff
@@ -18,7 +18,16 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: internal
+    impact: patch
```
Comment (alxsap) [inline changelog.yaml:23]: Are existing releases / changes not usable due to this "bug"?
Should we mark the old versions as buggy somehow?

Comment (alxsap) [reply changelog.yaml:23]: We might also track bugs with GitHub issues and add a field `issueNumber`?

File: changelog.yaml [hunk 3]
```diff
@@ -18,7 +18,16 @@
...
-  
+  changes:
+  - type: internal
+    impact: patch
+    title: Fix clustertask
+    description: >-
+      Fix variables in clustertask to changed structure.
+    warning:
+    upgradeNotes:
+
```
Comment (alxsap) [inline changelog.yaml:29]: remove the empty line

**Review by alxsap — APPROVED** [review]

File: changelog.yaml [hunk 4]
```diff
@@ -18,7 +18,16 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: internal
+    impact: patch
+    title: Fix clustertask
+    description: >-
+      Fix variables in clustertask to changed structure.
```
Comment (alxsap) [inline changelog.yaml:26]: The impact (or not) of this "bug" on older, released versions is unclear.

Comment (rinckm) [reply changelog.yaml:26]: As it is already required to upgrade tekton to v0.14.3 with steward version 0.5.0 there is no additional impact.

File: changelog.yaml (hunk 2 above)
Comment (rinckm) [reply changelog.yaml:23]: The old version is still supported by the latest tekton version.
It is just not allowed to mix old and new syntax.
https://github.com/tektoncd/pipeline/blob/v0.14.3/pkg/apis/pipeline/v1alpha1/task_validation.go#L65
https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1alpha1/task_validation.go#L66

File: changelog.yaml (hunk 3 above)
Comment (rinckm) [reply changelog.yaml]: Done

Comment (rinckm) [reply changelog.yaml]: Done

**Review by alxsap — APPROVED** [review]

**Pull request merged**